http/src/main/java/io/micronaut/http/annotation/Filter.java

```diff
@@ -68,4 +69,13 @@
      * @return The service identifiers
      */
     String[] serviceId() default {};
+
+
+    /**
+     * If provided, filter will be applied only to {@code io.micronaut.http.client.Client} that are marked
+     * with one of provided annotations.
+     *
+     * @return Marker annotations
+     */
+    Class<? extends Annotation>[] annotationMarkers() default {};
```

Comment:
To be consistent with how we name things in the user guide and the rest of the API I think this should be called `annotationStereotypes()` or just `stereotypes()`

Reply:
will rename to `stereotypes()`